pt-calculator-v2/src/main/java/org/egov/pt/calculator/web/controller/BillingSlabController.java

```diff
@@ -86,4 +92,56 @@
 		log.debug(" the time taken for create in ms: {}", endTime - startTime);
 		return new ResponseEntity<>(billingSlabRes, HttpStatus.OK);
 	}
+
+	/**
+	 * endpoint to create mutation billing-slabs
+	 *
+	 * @param mutationbillingSlabReq
+	 */
+	@PostMapping("/mutation/_create")
+	@ResponseBody
+	private ResponseEntity<?> mutationCreate(@RequestBody @Valid MutationBillingSlabReq mutationbillingSlabReq) {
+
+		long startTime = System.currentTimeMillis();
+		mutationbillingSlabValidator.validateCreate(mutationbillingSlabReq);
+		MutationBillingSlabRes billingSlabRes = mutationService.createBillingSlab(mutationbillingSlabReq);
+		long endTime = System.currentTimeMillis();
+		log.debug(" the time taken for create in ms: {}", endTime - startTime);
+		return new ResponseEntity<>(billingSlabRes, HttpStatus.CREATED);
+	}
+
+	/**
+	 * enpoint to update mutation billing-slabs
+	 *
+	 * @param billingSlabReq
+	 */
+	@PostMapping("/mutation/_update")
+	@ResponseBody
+	private ResponseEntity<?> update(@RequestBody @Valid MutationBillingSlabReq billingSlabReq) {
```

Comment:
Rename this method to `updateMutation`, there is already an `update` method used earlier